2022/Q2/artifacts/pos-model/PoS-model.md

```diff
@@ -106,13 +101,8 @@ tx_deactivate(validator_address)
 ```go
 tx_reactivate(validator_address)
 {
-  //VP:https://github.com/anoma/anoma/blob/master/proof_of_stake/src/validation.rs#L459
-  //https://github.com/anoma/anoma/blob/master/proof_of_stake/src/validation.rs#L471
-  var pre_state = read_epoched_field(validators[validator_address].state, cur_epoch)
-  var pre_state_offset = read_epoched_field(validators[validator_address].state, cur_epoch+pipeline_length)
-  if (pre_state == inactive && pre_state_offset in {pending, inactive}) then
-    //set status to pending inmediately
-    validators[validator_address].state[cur_epoch] = pending
+  var cur_state = read_epoched_field(validators[validator_address].state, cur_epoch+pipeline_length)
```

Comment:
I really like how the three actions related to validator state have been simplified, without losing power of expression